### PR TITLE
Bug Fix #341

### LINKED ISSUE #341: BUG: rejected mappings somehow fails.
```
make[2]: Leaving directory '/work/src/ontology'
test -f components/rejected-mappings.owl || touch components/rejected-mappings.owl
robot --catalog catalog-v001.xml convert -i components/rejected-mappings.owl -f json -o tmp/component-rejected-mappings.json
OBO GRAPH ERROR Could not convert ontology to OBO Graph (see https://github.com/geneontology/obographs)
For details see: http://robot.obolibrary.org/errors#obo-graph-error
Use the -vvv option to show the stack trace.
Use the --help option to see usage information.
make[1]: *** [mondo-ingest.Makefile:173: tmp/component-rejected-mappings.json] Error 1
rm imports/ro_terms_combined.txt
make[1]: Leaving directory '/work/src/ontology'
make: *** [mondo-ingest.Makefile:326: build-mondo-ingest] Error 2
Command exited with non-zero status 2
```

### PR DESCRIPTION
**Pull request by hrshdhgd**
Fixes #341

So we have make recipe : 
```
$(MAPPINGSDIR)/%.sssom.tsv: $(TMPDIR)/component-%.json metadata/mondo.sssom.config.yml
	sssom parse $< -I obographs-json --prefix-map-mode merged -m metadata/mondo.sssom.config.yml -o $@
	sssom sort $@ -o $@
```

which has nothing to do with rejected mappings. I had the target `$(MAPPINGSDIR)/rejected-mappings.sssom.tsv` which executed the above and hence the error seen in #341.

I renamed it to `$(MAPPINGSDIR)/rejected-mappings-sssom.tsv` and this should fix the problem.

### CODE REVIEW

File: src/ontology/metadata/mondo.sssom.config.yml
Comment (matentzn) [inline src/ontology/metadata/mondo.sssom.config.yml:1]: Are these changes intended?

Comment (hrshdhgd) [reply src/ontology/metadata/mondo.sssom.config.yml:1]: Fixed

**Review by matentzn — APPROVED** [review]
